manifests/REALiX/HWiNFO/6.26.yaml

```diff
@@ -0,0 +1,18 @@
+Id: REALiX.HWiNFO
+Publisher: HWiNFO
+Name: REALiX
```

Comment:
```suggestion
Publisher: REALiX
Name: HWiNFO
```

Reply:
Thank you! I will correct it as soon as possible.